src/06-loops.js

```diff
@@ -25,8 +31,15 @@ function rangeSum(min, max) {}
  *  rangeOdd(10, 15);
  *  //> [ 15, 13, 11 ]
  */
-function rangeOdd(min, max) {}
-
+function rangeOdd(min, max) {
+  let arr = [];
+  for (let i = max; i >= min; i--){
```

Comment:
We are iterating backwards here.  It's always a good idea to iterate forward unless we need to do otherwise.